README.md

```diff
@@ -202,6 +202,16 @@ Documentation for using each command is available by typing:
 > python manage.py help <command>
 ```
 
+### Pickle Tribe gene sets
+
+This step downloads and pickles all the public Tribe gene sets for every
+organism that has been loaded into the database. These pickled files are
+necessary for doing gene set enrichment analyses.
```

Comment:
@mhuyck : Ok, yeah, I agree, that sounds good and will be a quick change. I will insert a small hyperlink.